server/xml.server.php

```diff
@@ -91,6 +91,7 @@
     // If the method is the same as the action being called
     // Then let's call this function!
     if ($_GET['action'] == $method) {
+        $_GET['format'] = 'xml';
```

Comment:
Since the file names are different, its already decided by the user.

If you call xml.server.php you want XML
if you call json.server.php you want JSON